packages/data/addon/adapters/facts/elide.ts

```diff
@@ -87,7 +98,10 @@ export default class ElideFactsAdapter extends EmberObject implements NaviFactAd
         return null;
       }
 
-      const fieldStr = getElideField(field, parameters);
+      // TODO: Parameters cannot be specified in filters yet
```

Comment:
As discussed offline, let's throw an error if a parameter is set, until it is supported in elide